wp-parsely.php

```diff
@@ -268,13 +270,49 @@ public function initialize_settings() {
                 'help_text' => $h,
                 'requires_recrawl' => true));
 
+        // Allow use of custom taxonomy to populate articleSection in parselyPage; defaults to category
+        $h = 'By default, Parsely only tracks the default post type as a post page. ' .
+                'If you want to track custom post types, select them here!<br>';
+        add_settings_field('track_post_types',
```

Comment:
Wondering if we can be more explicit/less jargon-y here. Maybe say "tracks the 'default' WordPress post type as an article page, meaning that metadata values like title, author, etc. are displayed in the dashboard record. If you want to track custom WordPress post types as article pages as well..."

Thoughts? I feel like we can't assume too much Parse.ly knowledge for plugin users.